pkgs/tools/networking/openconnect/default.nix

```diff
@@ -1,6 +1,7 @@
-{ lib, stdenv, fetchurl, pkg-config, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, fetchgit, darwin } :
+{ lib, stdenv, fetchurl, pkg-config, openssl ? null, gnutls ? null, gmp, libxml2, p11-kit, libp11, stoken, zlib, fetchgit, darwin } :
```

Comment:
Replace `darwin` here with `PCSC`

Reply:
@roberth Could you add these changes?
@SuperSandro2000 I guess it makes more sense to first merge this pr before refactoring the `vpnc-script` into its own module as suggested https://github.com/NixOS/nixpkgs/pull/106465.

Reply:
I'm not using/maintaining smart card packages anymore. I think you meant @trobert :)

Reply:
@roberth Sorry about that and thanks for letting me know. I edited my comment.
At least I was close with the name confusion😃.

Reply:
Yeah, sure. We can do that.